src/wizard.scss

```diff
@@ -0,0 +1,300 @@
+  }
+
+  &__step-wrapper {
+    @include fd-reset();
+    @include fd-flex-vertical-center();
+
+    height: 2rem;
+  }
+
+  &__step {
+    @include fd-reset();
+
+    width: 100%;
+    min-width: 10.5rem;
+    position: relative;
+    display: table-cell;
+    vertical-align: top;
+
+    // --- STEP MODIFIERS ---
+    &--completed {
+      .#{$block}__step-indicator {
+        background: var(--sapObjectHeader_Background);
+        color: var(--sapWizard_Completed_Step_Color);
+        border: var(--sapWizard_Completed_Step_Border);
+
+        &::before {
+          color: var(--sapWizard_Completed_Step_Icon_Color);
+        }
+      }
+    }
+
+    &--current {
+      .#{$block}__step-indicator {
+        position: relative;
+        background: var(--sapSelectedColor);
+        color: var(--sapWizard_Current_Step_Color);
+
+        &::before {
+          color: var(--sapWizard_Current_Step_Icon_Color);
+        }
+
+        // Selection Bar
+        &::after {
+          content: "";
+          bottom: -1rem;
+          left: -$fd-wizard-step-indicator-spacing;
+          position: absolute;
+          display: inline-block;
+          width: calc(100% + 2 * 0.1875rem);
+          height: var(--sapWizard_Selection_Bar_Height);
+          background: var(--sapWizard_Selection_Bar_Background);
+          border-radius: 0.125rem 0.125rem 0 0;
+          transition: all $fd-animation-speed ease-in;
+        }
+      }
+    }
+
+    &--upcoming {
+      .#{$block}__step-indicator {
+        background: var(--sapObjectHeader_Background);
+        border: var(--sapWizard_Upcoming_Step_Border);
+        color: var(--sapWizard_Upcoming_Step_Color);
+
+        &::before {
+          color: var(--sapWizard_Upcoming_Step_Icon_Color);
+        }
+      }
+
+      .#{$block}__step-container {
+        cursor: default;
+        pointer-events: none;
+      }
+    }
+
+    &--no-label {
+      min-width: $fd-wizard-step-no-label-max-size;
+
+      .#{$block}__label-container {
+        display: none;
+      }
+    }
+
+    &--stacked-top {
+      &.#{$block}__step--stacked {
+        min-width: $fd-wizard-step-no-label-max-size;
+        width: $fd-wizard-step-no-label-max-size;
+
+        .#{$block}__connector {
+          display: block;
+          min-width: auto;
+        }
+      }
+    }
+
+    &--stacked {
+      width: 0.25rem;
+      min-width: auto;
+
+      .#{$block}__connector,
+      .#{$block}__label-container {
+        display: none;
+      }
+
+      &.#{$block}__step--upcoming {
+        &:last-child {
+          padding-right: $fd-wizard-stacked-upcoming-step-offset;
+
+          @include fd-rtl() {
+            padding-right: 0;
+            padding-left: $fd-wizard-stacked-upcoming-step-offset;
+          }
+        }
+
+        // Fiori 3 recommends a maximum of 8 steps
+        @for $i from 1 through 8 {
+          &:nth-child(#{$i}) {
+            z-index: calc(10 - #{$i});
```

Comment:
Great Job with it, but could you add more z-index for focused upcoming elements?
![wizard](https://user-images.githubusercontent.com/26483208/93209322-4122af00-f75e-11ea-9bcf-04f7f63e1842.gif)

Reply:
I don't understand how more z-index will solve the problem. Could you elaborate, please? The upcoming elements should not be focused. I disabled the pointer events but for tabs I need to use JavaScript so we will handle this in the implementation libraries.